config/example/admins.txt

```diff
@@ -5,4 +5,4 @@
 # Ranks can be anything defined in admin_ranks.txt           ~Carn   #
 ######################################################################
 
-# not_a_user - Admin
+Mycah - Host
```

Comment:
You changed the example.txt. You're supposed to drag this out of the example folder and add your name, not leave it in.

Reply:
I thought you said that would be ignored by the git. I did ask if I should remove the admin.txt file.

Reply:
Actual configs _are_ ignored, not folder with examples tho.